rogii/version.cmake

```diff
@@ -0,0 +1,19 @@
+set(
+    ECL_VERSION_MAJOR
```

Comment:
I think we should rename our variables because ecl uses its own ECL_VERSION_MAJOR for its own purposes.
https://github.com/rogii-com/ecl/blob/rogii-v2.7.0-dev/lib/CMakeLists.txt#L197-L199

Something like ROGII_ECL_VERSION_MAJOR